t/t2260-job-list.t

```diff
@@ -438,9 +438,9 @@ test_expect_success 'flux job list-inactive w/ since -1 leads to error' '
 	grep "Invalid argument" list_inactive_error1.out
 '
 
-test_expect_success 'flux job list-inactive w/ count -1 leads to error' '
+test_expect_success HAVE_JQ 'flux job list-inactive w/ count -1 leads to error' '
```

Comment:
We don't need `HAVE_JQ` anymore since b63aefdd5b20762e5fc05e52073ca16f7a9776cb.

But also, the test doesn't appear to use `jq` anyway?

Reply:
agh sorry, rebase error, this PR was originally from before that HAVE_JQ removal